### PR TITLE
Add core maintainers to website footer.

### LINKED ISSUE #414: [chore] Add core maintainers to footer
colorjs.io currently says this in the footer:

> From [Lea Verou](http://lea.verou.me/) (co-editor of CSS Color 4 and 5) and [Chris Lilley](https://svgees.us/) (co-editor of CSS Color 3, 4, and 5; W3C representative to ICC)

This is probably useful stuff, but it's not giving enough credit to core maintainers who have put a ton of work into Color.js.
I would change "From" to "Founded by" and add ". Co-maintained by [ list of maintainers ]".
This should basically be people senior enough that are org members, not just repo collaborators (maybe a year of consistent participation?).
We should also have something generally crediting the contributors, and linking to https://github.com/color-js/color.js/graphs/contributors

cc @svgeesus @jgerigmeyer @MysteryBlokHed

### PR DESCRIPTION
**Pull request by jgerigmeyer**
Closes #414.

I ordered maintainers by date of first commit to the repo.

cc @lloydk

### CODE REVIEW

**Review by MysteryBlokHed — APPROVED** [review]
LGTM

**Review by facelessuser — APPROVED** [review]

**Review by svgeesus — APPROVED** [review]

File: _includes/plain.njk [hunk 1]
```diff
@@ -26,8 +26,14 @@
...
 <footer>
-	From <a href="http://lea.verou.me">Lea Verou</a> (co-editor of CSS Color 4 and 5)
-	and <a href="https://svgees.us">Chris Lilley</a> (co-editor of CSS Color 3, 4, and 5; W3C representative to ICC)
+	Created by <a href="http://lea.verou.me">Lea Verou</a> (co-editor of CSS Color 4 and 5)
+	and <a href="https://svgees.us">Chris Lilley</a> (co-editor of CSS Color 3, 4, and 5; W3C representative to ICC).
+
+	Co-maintained by <a href="https://github.com/facelessuser">Isaac Muse</a>,
+	<a href="https://github.com/jgerigmeyer">Jonny Gerig Meyer</a>,
+	<a href="https://github.com/MysteryBlokHed">Adam Thompson-Sharpe</a>,
+	and <a href="https://github.com/lloydk">Lloyd Kupchanko</a>,
```
Comment (LeaVerou) [inline _includes/plain.njk:35]:
```suggestion
	and 
	<a href="https://github.com/MysteryBlokHed">Adam Thompson-Sharpe</a>,
```

**Review by LeaVerou — CHANGES_REQUESTED** [review]
Folks, while I was the one that suggested we invite @lloydk as a co-maintainer (and everyone enthusiastically agreed), this is a discussion that needed to happen *before* we add them to public facing things like these, this cannot be how they find out we’ve invited them 😅

Also, when I opened #414 I was envisioning adding the core maintainers, i.e. those who have been around a while and are also org members. @lloydk definitely has the volume and length of contribution for that, but I think it's still good practice to see how we all work together as a team first, even for a little bit.

**Review by LeaVerou — APPROVED** [review]

**Comment by LeaVerou** [comment]
Merged, thanks @jgerigmeyer!

**Pull request merged**